packages/react-native/README.md

```diff
@@ -0,0 +1,23 @@
+# Amplify UI
+
+Amplify UI is an open-source UI library with cloud-connected components that are endlessly customizable, accessible, and can integrate into _any_ application. Amplify UI consists of:
+
+1. Connected components that simplify complex cloud-connected workflows, like Authenticator.
+2. Primitive components that create consistency across Amplify UI and allow you to build complete applications that fit your brand, like Buttons and Badges.
+3. Data-bound components that make it easy to display dynamic data, like DataStoreCollections.
+4. Theming capabilities that allow you to customize the appearance of Amplify UI to match your brand.
+
+## React Native Documentation
+
+- https://ui.docs.amplify.aws/react-native
+
+## Features 🚀
+
+- **Better developer experience** Connected-components like Authenticator are being written with framework-specific implementations so that they follow framework conventions and are easier to integrate into your application.
+- **Endlessly customizable** Every detail of Amplify UI is customizable to match your brand. Style all of Amplify UI with themes, override components with your own, or build your own UI and use Amplify for complex state management.
+- **Accessible** Amplify UI components follow [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) and [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.2/) best practices and guidelines such as color contrast, keyboard navigation, accessible labels, and focus management.
+- **Primitive components (React only right now)** Primitive components are used in the connected components, like Authenticator, you can also customize them and use them to build the rest of your UI.
+
+## We love contributors!!
+
+See our contributing guide [CONTRIBUTING.md](/CONTRIBUTING.md) to help us scale Amplify UI!
```

Comment:
for later, it'd be nice to have a `CONTRIBUTING.md` for `react-native` like [how we do](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md) for `react`.